WebAPI/WebComponents/Meeting/src/AgendaItem.js

```diff
@@ -64,63 +71,30 @@ export default function AgendaItem(props) {
         }
     }
 
-    const editableDecision = () => {
+    const manageContent = (html) => {
         var div = document.createElement('div')
         var newDiv = document.createElement('div')
-        div.innerHTML = agenda.html
-        const newItem = div.querySelectorAll(".SisaltoSektio")[0]
-        if (newItem) {
-            var editableDiv = document.createElement('div')
-            editableDiv.innerHTML = div.querySelectorAll(".SisaltoPaatos")[0] || ""
-            return editableDiv.innerHTML
-        } else {
-            const nodes = div.childNodes
-            for (var i = 0; i < nodes.length; i++) {
-                if (nodes[i].nodeName == 'H1' || nodes[i].nodeName == 'H2' || nodes[i].nodeName == 'H3' || nodes[i].nodeName == 'H4') {
-                    var b = document.createElement('b')
-                    b.textContent = nodes[i].textContent
+        div.innerHTML = html
+        var section = div.querySelector(".SisaltoSektio") || div
+        var nodes = section.childNodes
+        for (var i = 0; i < nodes.length; i++) {
+            var element = nodes[i].cloneNode(true)
```

Comment:
could be const also?

Reply:
element is changed so has to be variable

Reply:
ok